dbt/adapters/databricks/connections.py

```diff
@@ -39,6 +40,7 @@
 logger = AdapterLogger("Databricks")
 
 CATALOG_KEY_IN_SESSION_PROPERTIES = "databricks.catalog"
+CUSTOM_USER_AGENT = "__DBT_DATABRICKS_CUSTOM_USER_AGENT__"
```

Comment:
Are there precedents for the naming convention used here?

Reply:
:+1: